streaming/rrdpush.c

```diff
@@ -90,13 +95,20 @@ int rrdpush_init() {
             }
         }
     }
+
     char *invalid_certificate = appconfig_get(&stream_config, CONFIG_SECTION_STREAM, "ssl skip certificate verification", "no");
     if ( !strcmp(invalid_certificate,"yes")){
         if (netdata_validate_server == NETDATA_SSL_VALID_CERTIFICATE){
-            info("The Netdata is configured to accept invalid certificate.");
+            info("The Netdata is configured to accept invalid SSL certificate.");
             netdata_validate_server = NETDATA_SSL_INVALID_CERTIFICATE;
         }
     }
+
+    netdata_ssl_cert_directory = appconfig_get(&stream_config, CONFIG_SECTION_STREAM, "CApath", "/etc/ssl/certs/");
+    netdata_ssl_master_cert = appconfig_get(&stream_config, CONFIG_SECTION_STREAM, "CAfile", "/etc/ssl/certs/");
+    if(!strcmp(netdata_ssl_cert_directory,netdata_ssl_master_cert)) {
```

Comment:
This isn't correct. Either of the two settings could actually be null.

Reply:
It was also fixed and the description for this is not different from the previous that I gave.

Reply:
Dir as default for CAFile?

> /etc/ssl/certs/

Since those parameters related to CA i think it is nice to have `ca` in the name.

Reply:
`CApath` and `CAfile` have same meaning, at least for me. `path` suffix doesnt mean dirtory. If we have `CAFile` then it feels natural to use `CADir`.

Reply:
> `CApath` and `CAfile` have same meaning, at least for me. `path` suffix doesnt mean dirtory. If we have `CAfile` then it feels natural to use `CAdir`.

I lost you a little bit here, those two variables mean exactly what they say.
CAPath -> The directory where multiple certificates on the right format may exist.
CAFile -> A single CA Certificate file path

Reply:
i am only saying that `CADir` looks better for me then `CAPath` since we are using `CAFile`. I admit that `CAPath` will do too.

Reply:
> Dir as default for CAFile?
> 
> > /etc/ssl/certs/
> 

Hi @ilyam8 , this was the first commit in this PR, it is not my this way since yesterday last commit. I used the fact that on Unices everything is a file

> Since those parameters related to CA i think it is nice to have `ca` in the name.
> 
> > netdata_ssl_cert_directory
> > netdata_ssl_master_cert

I liked your idea, it will be more readable.

>>i am only saying that CADir looks better for me then CAPath since we are using CAFile (it is obvious that both of them are paths). I admit that CAPath will do too.

I agree completely, I also prefer Directory instead Path, but case we do this we will lose the connection with the OpenSSL documentation. We renamed it to CApath and CAfile for our users to have more sources of information.

@paulkatsoulakis due the fact it was not commit, I executed Ilya's suggestion in  this PR yet. This PR also brings fixes for the 1.16.1, is it not better to merge it?